translations/Hindi/general-rules.md

```diff
@@ -0,0 +1,39 @@
+# General Rules for Translation in Hindi
+
+## Overview
+
+* Hindi is a very flexible & simple language. Since it is very much like English, the translations are an easy task.
+
+* Translations can be either formal or informal, but formal translations are preferred for corporate work.
+
+* The translations can be done in any person (first, second or third) according to the content one wants to translate.
```

Comment:
Terminate your lines at 80 characters. Using one newline continues the same sentence/paragraph in markdown, but has the advantage of being easy to read.

Reply:
Sure

Reply:
Terminated all lines except the links